pkg/manager/service.go

```diff
@@ -276,13 +304,18 @@ func (m *Manager) Sync(ctx context.Context, req *rpc.SyncRequest) (resp *rpc.Bac
 	if err != nil {
 		return nil, err
 	}
-	portReleaseChannel := make(chan interface{})
+	portReleaseChannel := make(chan interface{}, 1)
 	go func() {
 		<-portReleaseChannel
+		log.Infof("Backing Image Manager: start to release port %v after syncing", port)
+		if !util.DetectHTTPServerAvailability(fmt.Sprintf("http://localhost:%d", port), 180, false) {
```

Comment:
Right now it seems the best effort to release the used port, even the port actually somehow occupied after timeout.

nit: in the future, we should probably have a mechanism to reclaim the orphaned port instead of only removing it from the port usage info at runtime if the backing image manager pod does not be restarted yet.

Reply:
Later on, we may coalesce the sparse file transfer related APIs into the file-syncing server. Then the file-syncing server can handle the transferring of multiple files but on a single persistent port. In other words, we don't need to worry about the port allocation or release issue, at least for the backing image manager.